--- a/rocrail/makefile	2026-08-03 03:12:40.381054805 +0200
+++ b/rocrail/makefile	2026-08-03 03:12:40.381087826 +0200
@@ -196,8 +196,6 @@
 
 # for unix based systems only
 install_all:
-#	-rm -Rf $(DESTDIR)
-#	-mkdir -p ~$(FS)rocrail$(FS)locxpm
 	-mkdir -p $(DESTDIR)
 	-mkdir -p $(DESTDIR)$(FS)default
 	-cp -p $(OUTDIR)$(FS)rocrail$(BINSUFFIX) $(DESTDIR)
@@ -216,14 +214,6 @@
 ifeq (,$(wildcard $(DESTDIR)$(FS)default$(FS)plan.xml))
 	-cp -p$(CPUPDATE) package$(FS)plan.xml $(DESTDIR)$(FS)default
 endif
-	-cp -p package$(FS)rocrail.sh $(DESTDIR)
-	-cp -p package$(FS)rocview.sh $(DESTDIR)
-	-chmod +x $(DESTDIR)$(FS)rocrail.sh
-	-chmod +x $(DESTDIR)$(FS)rocview.sh
-	-cp -p package$(FS)Rocrail.desktop $(FS)usr$(FS)share$(FS)applications
-	-cp -p package$(FS)Rocview.desktop $(FS)usr$(FS)share$(FS)applications
-	-cp -p package$(FS)Rocrail.directory $(FS)usr$(FS)share$(FS)desktop-directories
-#	-chmod -R go+rw ~$(FS)rocrail
 
 uninstall_all:
 	-$(REMOVE) -Rf $(DESTDIR)
